src/Handlers/PaginatedPageHandler.php

```diff
@@ -36,7 +36,7 @@ public function handle($file, PageData $pageData)
 
         return $this->paginator->paginate($file,
             $pageData->get($pageData->page->pagination->collection),
-            $pageData->page->pagination->perPage ?: 10
+            $pageData->page->pagination->perPage ?: ($pageData->page->default_perPage ?: 1)
```

Comment:
- Please change `default_perPage` to `perPage`
- Update the fallback default from `1` to `10`